db/migrate/20240404142248_create_aliquots.rb

```diff
@@ -0,0 +1,23 @@
+class CreateAliquots < ActiveRecord::Migration[7.0]
+  def up
+    create_table :aliquot, primary_key: :id, options: 'CHARSET=utf8 COLLATE=utf8_unicode_ci' do |t|
+      t.string :id_lims, null: false, comment: 'The LIMS system that the aliquot was created in'
+      t.string :lims_uuid, null: false, comment: 'The UUID of the aliquot in the LIMS system'
+      t.string :aliquot_type, null: false, comment: 'The type of the aliquot'
+      t.string :source_type, null: false, comment: 'The type of the source of the aliquot'
+      t.string :source_barcode, null: false, comment: 'The barcode of the source of the aliquot'
+      t.string :sample_name, null: false, comment: 'The name of the sample that the aliquot was created from'
+      t.string :used_by_type, null: false, comment: 'The type of the entity that the aliquot is used by'
+      t.string :used_by_barcode, null: false, comment: 'The barcode of the entity that the aliquot is used by'
+      t.decimal :volume, precision: 10, scale: 2, null: false, comment: 'The volume of the aliquot'
+      t.decimal :concentration, precision: 10, scale: 2, null: false, comment: 'The concentration of the aliquot'
```

Comment:
@BenTopping 
Just checking whether you are storing concentration or molarity here and if the units are always the same?
Concentrations are typically in ng/ul (nanograms per microlitre). Molarity is usually in nM (nanomolar). But not always.
In Sequencescape this is done as 2 fields, a key ('concentration' or 'molaity') and a units ('ng/ul', 'g/ml', 'nM', 'M', etc.)

Reply:
If fixed put the expected units in the comment?

Reply:
We are storing concentration. I have spoken to James Watts and this is deemed useful for calculations. Volume will be uL and concentration ng/ul. As there is no intention to reuse for short read currently we can leave as is.

Reply:
@stevieing I will add them in the column comments (uL in `volume` column's comment and ng/ul in `concentration` column's comment).